runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -1502,9 +1510,24 @@ TR_VectorAPIExpansion::isVectorizedOrScalarizedNode(TR::Node *node, TR::DataType
          {
          ncount_t nodeIndex = node->getGlobalIndex();
 
-         elementType = _nodeTable[nodeIndex]._elementType;
-         bitsLength = _nodeTable[nodeIndex]._vecLen;
          objectType = _nodeTable[nodeIndex]._objectType;
+
+         TR::MethodSymbol *methodSymbol = origSymRef->getSymbol()->castToMethodSymbol();
+
+         if (sourceType &&
+             methodSymbol->getRecognizedMethod() == TR::jdk_internal_vm_vector_VectorSupport_convert)
+            {
+            // source type of two-type opcodes is not stored in the _nodeTable
+            // so we need to analyze the node again
+            bool result = getConvertSourceType(this, node, elementType, bitsLength);
```

Comment:
If `node` has already been transformed, then will it be in the form required for `getConvertSourceType()`?

Reply:
Tests passed so I would keep that assert.